### DISCUSSION TITLE
gcn-1.0.0-snapshot-08042024 is an internal, temporal snapshot

### DISCUSSION THREAD

**Discussion by dolukhanov** [spring-projects/spring-ai#567] [Q&A] [answered]
I saw @tzolov released [gcn-1.0.0-snapshot-08042024](https://github.com/spring-projects/spring-ai/releases/tag/gcn-1.0.0-snapshot-08042024) yesterday, which now includes direct Anthropic Claude 3 support.

Is this an 'official' release - or should we wait for v0.8.2?

**Comment by tzolov** [comment, marked as answer]
No, the `gcn-1.0.0-snapshot-08042024` is a temporal, internal (snapshot tag as the name snapshot suggests) and will be deleted soon!

The next milestone `1.0.0-M1` (after current 0.8.1 milestone) is coming hopefully next week. We intend to have a 10.0.RC1 and GA in couple of weeks after.  @markpollack does this make sense? 

@dolukhanov i've rewrote the title to avoid confusion. 

Current, 1.0.0-SNAPSHOT already offers full support for:

- direct Anthropic 3 (including streaming and multimodality): https://docs.spring.io/spring-ai/reference/1.0-SNAPSHOT/api/chat/anthropic-chat.html
  - Function calling: https://docs.spring.io/spring-ai/reference/1.0-SNAPSHOT/api/chat/functions/anthropic-chat-functions.html
- Bedrock Anthropic 3 (including streaming and multimodality): https://docs.spring.io/spring-ai/reference/1.0-SNAPSHOT/api/chat/bedrock/bedrock-anthropic3.html

**Comment by dolukhanov** [comment]
Hi @tzolov - I wanted to see if there was a target date for the 1.0.0-M1 release? Thanks
